assets/js/hooks/audio_player.js

```diff
@@ -243,9 +243,14 @@ AudioPlayer = {
     const node = document.getElementById(targetDomId)
     classVals.forEach(classVal => node.classList.add(classVal))
   },
-
   emitMediaBridgeJSUpdate(key, value, extraKey = "innerText") {
-    this.pushEventTo("#MediaBridge", "update_display_value", [key , value, extraKey])
+    const customEvent = new CustomEvent("update_display_value", {
+      bubbles: true,
+      detail: {payload: [key, value, extraKey]},
```

Comment:
note: the detail field of an event seems to be the correct way of passing msgs / args around